README.md

```diff
@@ -1 +1,72 @@
-# github-lessons
\ No newline at end of file
+# github-lessons
+
+## Types of Branches/Commits
+* Feature (new functionality)
+* Fix (bug fixes)
+* Refactor
+* Chore (Dependecy addition or project related commits)
+## Commands
+
+Get origin changes
+```
+git pull
+```
+
+Get origin branches
+```
+git fetch
+```
+
+Create a Branch  
+```
+git branch feature/name-of-the-branch
+```
+
+Checkout a branch
+```
+git checkout feature/name-of-the-branch
+```
+
+Push a new branch to origin
+```
+git push --set-upstream origin feature/name-of-the-branch
```

Comment:
This line needs refactoring. Please check this for cleaner code. For example you can this:

```
git branch feature/bla
```

Reply:
You're nuts that's not how we do it